gtests/net/mptcp/common/multi-ep.sh

```diff
@@ -0,0 +1,92 @@
+#!/bin/bash
+#
+# Assign additional (advertised) addresses to the local device,
+# add a local/remote subflow for each address
+
+usage() { echo "$0 [-e <endpoints>] [-m <signal|subflow>] [-b]" 1>&2; exit 1; }
+
+create_endpoints() {
+    local host= flags= ep=1 max=$1 hn=$2 nn=$3 pl=$4 sig=${5:-0} bck=${6:-0}
+
+
+    [ $sig -eq 1 ] && flags=${flags:+$flags }signal || flags=${flags:+$flags }subflow
+    [ $bck -eq 1 ] && flags=${flags:+$flags }backup
```

Comment:
(detail: maybe clearer with arrays but it also works like that)

Reply:
ok, but  concatenating the flags we want to set on the endpoint is sufficient

Reply:
> ok, but concatenating the flags we want to set on the endpoint is sufficient

Indeed, it is sufficient.

I find arrays in this case clearer:

```bash
[ $sig -eq 1 ] && flags=(signal) || flags=(subflow)
[ $bck -eq 1 ] && flags+=(backup)

(...)

ip mptcp endpoint add $host "${flags[@])"
```

but I'm fine without :)